docs/3.0rc/develop/write-tasks/use-task-runners.mdx

```diff
@@ -138,67 +127,47 @@ Each flow can only have a single task runner, but sometimes you may want a subse
 using a specific task runner. In this case, you can create [subflows](/3.0rc/develop/write-workflows/#composing-flows) 
 for tasks that need to use a different task runner.
 
-For example, you can have a flow (in the example below called `sequential_flow`) that runs its tasks locally 
-using the `SequentialTaskRunner`. If you have some tasks that can run more efficiently in parallel on a Dask cluster, 
+For example, you can have a flow (in the example below called `multiple_runner_flow`) that runs its tasks locally 
+using the `ConcurrentTaskRunner`. If you have some tasks that can run more efficiently in parallel on a Dask cluster, 
 you can create a subflow (such as `dask_subflow`) to run those tasks using the `DaskTaskRunner`.
 
 ```python
 from prefect import flow, task
-from prefect.task_runners import SequentialTaskRunner
+from prefect.task_runners import ConcurrentTaskRunner
 from prefect_dask.task_runners import DaskTaskRunner
+import time
 
 @task
 def hello_local():
+    time.sleep(2)
     print("Hello!")
 
+@flow(task_runner=ConcurrentTaskRunner())
+def concurrent_subflow():
+    hello_local.submit()
+    hello_local.submit()
+
 @task
 def hello_dask():
     print("Hello from Dask!")
 
-@flow(task_runner=SequentialTaskRunner())
-def sequential_flow():
-    hello_local.submit()
-    dask_subflow()
-    hello_local.submit()
-
 @flow(task_runner=DaskTaskRunner())
 def dask_subflow():
     hello_dask.submit()
 
+@flow(task_runner=ConcurrentTaskRunner())
```

Comment:
I wonder if this example should be removed, since the section was originally about running tasks sequentially, and this example doesnt demonstrate that anymore